hazelcast/src/main/java/com/hazelcast/map/AbstractEntryProcessor.java

```diff
@@ -65,10 +67,21 @@ public AbstractEntryProcessor(boolean applyOnBackup) {
         return entryBackupProcessor;
     }
 
-    private class EntryBackupProcessorImpl implements EntryBackupProcessor<K, V> {
+    private class EntryBackupProcessorImpl implements EntryBackupProcessor<K, V>, HazelcastInstanceAware {
```

Comment:
"This will change the serialVersionUID" <-- that's the reason this PR adds an explicit `serialVersionUID`. which happens to be the same as the generated `serialVersionUID` which does not implement the `HazelcastInstananceAware` interface.